charts/astronomer/templates/houston/houston-configmap.yaml

```diff
@@ -178,6 +178,7 @@ data:
         {{- if .Values.global.dagOnlyDeployment.securityContexts }}
         securityContexts: {{ template "dagOnlyDeployment.securityContexts" . }}
         {{- end }}
+        {{- if or .Values.global.dagOnlyDeployment.resources .Values.global.dagOnlyDeployment.server.livenessProbe .Values.global.dagOnlyDeployment.server.readinessProbe }}
```

Comment:
Nit: I wonder if wide helm template code like would be more readable if we we spanned it across multiple lines... 🤔

```
        {{- if or
          .Values.global.dagOnlyDeployment.resources
          .Values.global.dagOnlyDeployment.server.livenessProbe
          .Values.global.dagOnlyDeployment.server.readinessProbe
        }}
```

What do you think? I'm not sure it makes it more readable.

Reply:
Anything is okay as long as formatting does put back to earlier state